debian/changelog

```diff
@@ -205,7 +216,7 @@ libfm-qt (0.14.1-12ubuntu1) focal; urgency=medium
         trusting executables.
     - debian/patches: fix-dnd-mtp.patch
       + Fix drag-and-drop from mtp:// mounted folders.
-  
+
```

Comment:
Thank you for doing wrap-and-sort. Fun fact: the only time I would never run that command is for anything that goes into the archive for a stable release. The processes are rigorous and that would be fair game for criticism. Just an FYI :smile:

Reply:
Oh great. Alright, will learn to stop running that command before every. Single. Commit. When doing backports of archive stuff. (That's what I've been doing, didn't realize that would be a problem...)